test/dhcp/server_ms_test.rb

```diff
@@ -14,121 +14,116 @@ def setup
+      "    172.29.205.35     -    00-1e-68-04-a0-eb-",
+      "    172.29.205.36     -    00-1e-68-04-a7-0f-",
+      "    172.29.205.38     -    00-1e-68-04-a0-db-",
+      "    172.29.205.39     -    00-1e-68-04-a4-17-",
+      "    172.29.205.44     -    00-14-4f-41-09-90-",
+      "    172.29.205.45     -    00-14-4f-40-ea-98-",
+      "    172.29.205.52     -    00-14-4f-41-0b-74-",
+      "    172.29.205.53     -    00-14-4f-40-67-24-",
+      "    172.29.205.54     -    00-14-4f-40-db-80-",
+      "    172.29.205.55     -    00-14-4f-40-e1-f0-",
+      "    172.29.205.58     -    00-14-4f-40-e2-54-",
+      "    172.29.205.79     -    00-1b-24-1d-e6-60-",
+      "    172.29.205.78     -    00-1b-24-1d-e6-c8-",
+      "    172.29.205.77     -    00-1b-24-1d-e8-e0-",
+      "    172.29.205.80     -    00-1b-24-5b-e0-62-",
+      "    172.29.205.81     -    00-1b-24-93-34-39-",
+      "    172.29.205.82     -    00-04-23-dd-29-98-",
+      "    172.29.205.63     -    00-14-4f-41-09-cc-",
+      "    172.29.205.5      -    01-01-01-00-00-05-",
+      "    172.29.205.40     -    00-14-4f-41-0b-70-",
+      "    172.29.205.43     -    00-14-4f-41-09-c0-",
+      "    172.29.205.28     -    01-01-01-00-02-08-",
+      "    172.29.205.85     -    00-21-28-57-42-ce-",
+      "    172.29.205.86     -    00-21-28-57-41-c2-",
+      "    172.29.205.87     -    00-21-28-57-44-2a-",
+      "    172.29.205.88     -    00-14-4f-ca-ae-c8-",
+      "    172.29.205.89     -    00-21-28-57-40-e2-",
+      "    172.29.205.90     -    00-14-4f-ca-6a-70-",
+      "    172.29.205.70     -    00-21-28-57-41-72-",
+      "    172.29.205.31     -    99-10-10-11-12-12-",
+      "    172.29.205.10     -    00-1b-24-1d-e9-e4-",
+      "    172.29.205.57     -    00-14-4f-41-0a-18-",
+      "    172.29.205.76     -    00-e0-81-bb-05-18-",
+      "    172.29.205.71     -    00-14-4f-ca-91-48-",
+      "    172.29.205.91     -    00-e0-81-d2-ea-12-",
+      "    172.29.205.92     -    00-e0-81-d2-ea-06-",
+      "    172.29.205.93     -    00-e0-81-d2-01-aa-",
+      "    172.29.205.94     -    00-e0-81-d1-fe-2a-",
+      "    172.29.205.95     -    00-e0-81-d2-e9-aa-",
+      "    172.29.205.20     -    00-0c-29-e4-cd-0c-",
+      "    172.29.205.21     -    00-0c-29-3b-ec-dc-",
+      "    172.29.205.22     -    00-0c-29-cf-00-45-",
+      "    172.29.205.24     -    00-0c-29-e9-83-09-",
+      "    172.29.205.96     -    00-e0-81-d2-e9-56-",
+      "    172.29.205.6      -    00-14-4f-82-9c-ea-",
+      "    172.29.205.27     -    00-03-ba-45-79-31-",
+      "    172.29.205.23     -    00-0c-29-f6-f9-ce-",
+      "    172.29.205.25     -    00-0c-29-64-a1-f1-",
+      "    172.29.205.26     -    00-0c-29-46-8f-10-",
+      "",
+      "",
+      "No of ReservedIPs : 57 in the Scope : 172.29.205.0.",
+      "",
+      "Command completed successfully."])
+    @server.stubs(:execute).with("scope 172.29.205.0 Show OptionValue", "Queried 172.29.205.0 options").returns(["",
```

Comment:
@witlessbird I am not sure on this one. Currently I think GitHub shows it wrong it seems; but I think this needs also be an array; since it uses `server.stubs(:execute)`?